soccer/gameplay/plays/sequences/move_sequence.py

```diff
@@ -0,0 +1,22 @@
+import behavior_sequence
+import skills.move
+
+class MoveSequence(behavior_sequence.BehaviorSequence):
+    def __init__(self, positions = None):
+        super().__init__()
+        self._positions = [] if positions == None else positions
+
+    def on_enter_start(self):
+        super().on_enter_start()
+        for pos in self.positions:
+            self.behaviors.append(skills.move.Move(pos))
+
+    @property
```

Comment:
Ok, I've pinned down the problem. In your super call in the constructor, we enter the behaivor_sequence constructor, and the behavior_sequence constructor sets this play to the start state, which calls our overloaded function, which reads the value before we set it. We can fix this by not having an `on_enter_start` and instead making a transition when we start the play up (maybe on_enter_running?) I don't think we should ever have an `on_enter_start` in a play.